src/gui/fspathedit_p.cpp

```diff
@@ -80,6 +255,33 @@ void Private::FileLineEdit::keyPressEvent(QKeyEvent *e)
         m_completerModel->setRootPath(QFileInfo(text()).absoluteDir().absolutePath());
         showCompletionPopup();
     }
+
+    const FileSystemPathValidator *validator =
+        dynamic_cast<const FileSystemPathValidator *>(this->validator());
```

Comment:
In the very same way I always think "why should we use something else when C++ itself has it?" :)